hypothesis-python/src/hypothesis/internal/filtering.py

```diff
@@ -78,10 +78,10 @@ def get_numeric_predicate_bounds(predicate: Predicate) -> ConstructivePredicate:
     """
     if (
         type(predicate) is partial
-        and len(predicate.args) == 1
-        and not predicate.keywords
+        and len(predicate.args) == 1  # type: ignore
+        and not predicate.keywords  # type: ignore
```

Comment:
Does this work better if we write `isinstance(predicate, partial)`?  If not IMO we should probably open an issue for Mypy to support that.

I can imagine that we'd have to annotate `Predicate = Union[partial, Callable[..., Any]]`, which is silly but understandably low priority.

Reply:
Great idea! It works!